### DISCUSSION TITLE
Cannot run openxr with Pimax crystal

### DISCUSSION THREAD

**Discussion by Nater1977** [praydog/REFramework#949] [General Questions]
I’ve been trying every which way from Sunday to get RE4 running with Pimax crystal in open XR. Is there anything I’m missing? I have open xr tool kit running open composite, but I get errors, and I also have Pimax companion app
As soon as the game starts to load right when it tries the openXR the game just shut down
I tried running Pimax open XR in steam setting as well on the companion app, don’t know where to start next any help would be greatly appreciated.

**Comment by praydog** [comment]
OpenXR Toolkit can crash the game, and there is no reason to use OpenComposite. Just delete the openvr_api.dll that comes with the mod to use OpenXR

**Reply by Nater1977** [reply]
That’s what I did in the beginning and it did not work. so I figured I had to use all these other apps to get it running. I’m gonna start from scratch and see what I can do. Thanks for the reply.

**Reply by Nater1977** [reply]
Thanks Its All set ya the openxr was crashing it a lot i have it dialed in!

**Comment by Nater1977** [comment]
We are up and running! Thanks Praydog!
